worker/deploy/enhanced-worker.js

```
@@ -0,0 +1,879 @@
+    try {
+      switch (url.pathname) {
+        case '/health':
+          return new Response(JSON.stringify({ 
+            status: 'ok', 
+            timestamp: new Date().toISOString() 
+          }), {
+            headers: { 'Content-Type': 'application/json' }
+          });
+
+        case '/stats':
+          const stats = await worker.getStats();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)